apps/api/src/app/events/e2e/trigger-event.e2e.ts

```diff
@@ -2208,18 +2206,13 @@ describe(`Trigger event - ${eventTriggerPath} (POST)`, function () {
       });
     });
   });
-  describe.skip('Trigger Event - [IS_MULTI_PROVIDER_CONFIGURATION_ENABLED=true]', function () {
+  describe('Trigger Event - Multi-Provider Configuration', function () {
     beforeEach(async () => {
-      process.env.IS_MULTI_PROVIDER_CONFIGURATION_ENABLED = 'true';
       process.env.LAUNCH_DARKLY_SDK_KEY = '';
       session = new UserSession();
       await session.initialize();
     });
 
-    afterEach(async () => {
-      process.env.IS_MULTI_PROVIDER_CONFIGURATION_ENABLED = ORIGINAL_IS_MULTI_PROVIDER_CONFIGURATION_ENABLED;
-    });
-
     it('should trigger message with override integration identifier', async function () {
```

Comment:
You can move the test inside of the general describe as this describe is not needed anymore because of the removal of the feature flag.

Reply:
Good catch! Updated 👍